CRM/Activityreport/Data.php

```diff
@@ -26,30 +26,24 @@ public static function get($offset = 0, $limit = 0, $multiValuesOffset = 0) {
     self::$fields = self::getActivityFields();
     self::$emptyRow = self::getEmptyRow();
     self::$multiValues = array();
-    $cacheKey = 'get-' . $offset . ',' . $limit . ',' . $multiValuesOffset;
-
-    $result = CRM_Core_BAO_Cache::getItem('activityreport', $cacheKey);
-    if (empty($result)) {
-      $params = array(
-        'sequential' => 1,
-        'is_current_revision' => 1,
-        'is_deleted' => 0,
-        'is_test' => 0,
-        'return' => implode(',', array_keys(self::$fields)),
-        'options' => array(
-          'sort' => 'activity_date_time DESC',
-          'offset' => $offset,
-          'limit' => $limit,
-        ),
-      );
 
-      $activities = civicrm_api3('Activity', 'get', $params);
+    $params = array(
```

Comment:
Yeah, cache was removed intentionally as it's going to be implemented in another phase (v3) after everything other is finished.